src/components/atoms/Banner.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div class="bg-banner-img bg-cover p-10">
+    <div class="mx-auto container px-3">
+      <h1
+        class="
+          text-3xl
+          sm:text-4xl
+          px-3
+          py-1
+          font-bold font-heading
+          uppercase
+          bg-blue-deep bg-opacity-95
+          text-white
+          inline-block
+          whitespace-normal
+        "
+      >
+        {{ title }}
+      </h1>
+      <div class="">

```

Comment:
Can we remove this div and add CSS to the p tag instead?

Reply:
i took out the empty class but can't get rid of the div.